### PR TITLE
KAA-1178: Cluster setup guide - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Administration-guide/System-installation/Cluster-setup/index.md [hunk 1]
```diff
@@ -10,55 +10,53 @@ sort_idx: 30
...
+To refresh the key files, stop the kaa-node service, replace the key files, and start the service again.
 
-MariaDB come with out-of-the-box master-master replication support thus we recommend to use this database in your Kaa cluster. 
-You can find detailed instructions in [MariaDB cluster setup guide]({{root_url}}Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/).
+The Kaa node service requires some third party dependencies, such as SQL and NoSQL databases, and Apache Zookeeper service.
+For more information, see [Architecture overview]({{root_url}}Architecture-overview/).
+A choice of databases (for example, MongoDB + PostgreSQL) depends on your particular use case.
+For database cluster installation, refer to the official [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html), [MongoDB](https://docs.mongodb.com/manual/) and [PostgreSQL](https://www.postgresql.org/docs/) documentation.
```
Comment (sashadidukh) [inline doc/Administration-guide/System-installation/Cluster-setup/index.md]: In our Single node installation guide we use official Apache Cassandra database, so here we should change [Cassandra](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html) to [Cassandra](http://cassandra.apache.org/).

File: doc/Administration-guide/System-installation/Cluster-setup/index.md [hunk 2]
```diff
@@ -10,55 +10,53 @@ sort_idx: 30
...
 
-> Note: In a cluster you need to connect to databases from external hosts so you need to allow such external connections in corresponding database security configurations and configure firewall rules for database host machine. 
-> Refer to official documentation for corresponding database for security configuration details.
+MariaDB comes with the master-master replication support out of the box, therefore it is recommended that you use this database in your Kaa cluster.
```
Comment (sashadidukh) [inline doc/Administration-guide/System-installation/Cluster-setup/index.md]: MariaDB doesn't come with the master-master replication support out of the box. We should remove this sentence.

**Review by sashadidukh — CHANGES_REQUESTED** [review]

File: doc/Administration-guide/System-installation/Cluster-setup/index.md [hunk 3]
```diff
@@ -10,55 +10,53 @@ sort_idx: 30
...
+See [MariaDB cluster setup guide]({{root_url}}Administration-guide/System-installation/Cluster-setup/MariaDB-cluster-setup-guide/).
 
-In addition to added on ```kaa-node``` service installation step firewall rules on every endpoint Kaa Administrator need to add some more rules for databases (depending on a set databases in the cluster) and Zookeeper ports.
+To allow connections to the databases from external hosts, configure the corresponding database security settings and set up the firewall rules for the database host machine.
+For security configuration settings, refer to the database official documentation.
 
-> Note: This configuration will not affect AWS deployment or any other cloud provider and must be applied only if you setting up cluster on VMs or separate Linux machines. 
-> Next instructions depend on specific cloud provider. 
+In addition to the firewall rules set up during the Kaa node service installation process, you need to add some rules for the databases (depending on the choice of databases in the cluster) and Zookeeper ports.
```
Comment (sashadidukh) [inline doc/Administration-guide/System-installation/Cluster-setup/index.md]: Please add:
... you need to add some rules for the databases (depending on the choice of databases in the cluster), Kaa node thrift and Zookeeper ports.

File: doc/Administration-guide/System-installation/Cluster-setup/index.md [hunk 4]
```diff
@@ -104,18 +102,19 @@ $ sudo service iptables-persistent start
 $ sudo service iptables-persistent save
```
Comment (sashadidukh) [inline doc/Administration-guide/System-installation/Cluster-setup/index.md:105]: Please add before # Zookeeper port section # Kaa node thrift section:

# Kaa node thrift
$ sudo iptables -I INPUT -p tcp -m tcp --dport 9090 -j ACCEPT
$ sudo iptables -I OUTPUT -p tcp -m tcp --dport 9090 -j ACCEPT

**Review by sashadidukh — CHANGES_REQUESTED** [review]

File: doc/Administration-guide/System-installation/Cluster-setup/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/index.md]: Fixed.

File: doc/Administration-guide/System-installation/Cluster-setup/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/index.md]: Done.

File: doc/Administration-guide/System-installation/Cluster-setup/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/index.md]: Done.

File: doc/Administration-guide/System-installation/Cluster-setup/index.md (hunk 4 above)
Comment (hunch7) [reply doc/Administration-guide/System-installation/Cluster-setup/index.md:105]: Done.

**Comment by hunch7** [comment]
New PR: https://github.com/kaaproject/kaa/pull/1119

**Pull request closed without merging**